### ISSUE TITLE
How to use mmcv_custom

### ISSUE THREAD

**Issue by yuhua666** [ViTAE-Transformer/Remote-Sensing-RVSA#54]
I found an error while using your code that some modules were not registered. This is due to the unsuccessful installation of mmcv_custom. How can I correctly configure the environment using mmcv_custom?

**Comment by DotWang** [comment]
@yuhua666 import mmcv_custom in train.py

**Comment by yuhua666** [comment]
How should I modify train.exe now? Can you provide more details?

# Copyright (c) OpenMMLab. All rights reserved.
import argparse
import copy
import os
import os.path as osp
import time
import warnings

import mmcv
import torch
import torch.distributed as dist
from mmcv import Config, DictAction
from mmcv.runner import get_dist_info, init_dist
from mmcv.utils import get_git_hash
from mmdet import __version__
from mmdet.apis import init_random_seed, set_random_seed

from mmrotate.apis import train_detector
from mmrotate.datasets import build_dataset
from mmrotate.models import build_detector
from mmrotate.utils import collect_env, get_root_logger, setup_multi_processes

Also, don't I need to manually install the mmcv full package? I previously installed mmcv full 1.6.1 automatically

**Comment by yuhua666** [comment]
I used import mmcv_custom, but still encountered the same error. In addition, there are the following errors (deleting the corresponding line content can run normally, but I don't know if it will affect the training results)

DOTADataset: PolyRandomRotate: __init__() got an unexpected keyword argument 'angle_range'

**Comment by yuhua666** [comment]
It is the LayerDecayOptimizerConstructor_ViT class, which can be run by deleting the corresponding one from the configuration file, but I feel it will affect the accuracy.
